vdr/ambassador.go

```diff
@@ -78,16 +78,20 @@ func NewAmbassador(networkClient network.Transactions, didStore didstore.Store,
 	}
 }
 
-// Configure instructs the ambassador to start receiving DID Documents from the network.
```

Comment:
I found the symmetry this had with the VCR easier to understand. Should this be changed in the VCR too?

Reply:
Or add a comment explaining why subscribing happens in Start instead of Configure

Reply:
I'd prefer changing it in VCR as well

Reply:
The symmetry gives you an incorrect 'feeling' that both engines are the same. They aren't, the network is dependant on DIDs, not on VCs.

Reply:
Added comment

Reply:
We probably don't need `Configure` in ambassadors at all, after this change. But for the sake of progress let's say out of scope for now and fix later.